pkg/keyspace/keyspace.go

```diff
@@ -49,10 +49,9 @@ const (
 	UserKindKey = "user_kind"
 	// TSOKeyspaceGroupIDKey is the key for tso keyspace group id in keyspace config.
 	TSOKeyspaceGroupIDKey = "tso_keyspace_group_id"
-	// keyspacePatrolBatchSize is the batch size for keyspace assignment patrol.
-	// the limit of etcd txn op is 128, keyspacePatrolBatchSize need to be less than it.
+	// maxTxnOps is the batch size for operating etcd. The limit of etcd txn op is 128.
 	// See: https://github.com/etcd-io/etcd/blob/d3e43d4de6f6d9575b489dd7850a85e37e0f6b6c/server/embed/config.go#L61
-	keyspacePatrolBatchSize = 120
+	maxTxnOps = 120
```

Comment:
The batch size is determined for the loaded keyspace, but in fact other operators may be included in the PatrolKeyspaceAssignment, such as SaveKeyspaceMeta and SaveKeyspaceGroup and LoadKeyspaceGroup.

Reply:
What do you mean here?

Reply:
maxTxnOps is not suitable than keyspacePatrolBatchSize

Reply:
not only Patrol uses this value